dataland-backend/src/test/kotlin/org/dataland/datalandbackend/service/DataManagerTest.kt

```diff
@@ -16,42 +19,57 @@ class DataManagerTest(
 
     val testManager = DataManager(edcClient, objectMapper)
 
-    val testCompanyNamesToStore = listOf("Imaginary-Company_I", "Fantasy-Company_II", "Dream-Company_III")
+    val testCompanyList = listOf(
+        CompanyInformation(
+            companyName = "Test-Company_1",
+            headquarters = "Test-Headquarters_1",
+            industrialSector = "Test-IndustrialSector_1",
+            marketCap = BigDecimal(100),
+            reportingDateOfMarketCap = LocalDate.now()
+        ),
+        CompanyInformation(
+            companyName = "Test-Company_2",
+            headquarters = "Test-Headquarters_2",
+            industrialSector = "Test-IndustrialSector_2",
+            marketCap = BigDecimal(200),
+            reportingDateOfMarketCap = LocalDate.now()
+        )
+    )
 
     @Test
     fun `add the first company and check if its name is as expected by using the return value of addCompany`() {
-        val companyMetaInformation = testManager.addCompany(testCompanyNamesToStore[0])
+        val companyMetaInformation = testManager.addCompany(testCompanyList[0])
         assertEquals(
-            companyMetaInformation.companyName, testCompanyNamesToStore[0],
+            companyMetaInformation.companyInformation.companyName, testCompanyList[0].companyName,
             "The company name in the post-response does not match the actual name of the company to be posted."
         )
     }
 
     @Test
     fun `add all companies then retrieve them as a list and check for each company if its name is as expected`() {
-        for (companyName in testCompanyNamesToStore) {
-            testManager.addCompany(companyName)
+        for (company in testCompanyList) {
+            testManager.addCompany(company)
         }
 
         val allCompaniesInStore = testManager.listCompaniesByName("")
         for ((counter, storedCompany) in allCompaniesInStore.withIndex()) {
             assertEquals(
-                testCompanyNamesToStore[counter], storedCompany.companyName,
+                testCompanyList[counter].companyName, storedCompany.companyInformation.companyName,
```

Comment:
wie oben, Prüfung des Namens allein erscheint mir nicht ausreichend.

Reply:
Ich habe die Tests in dieser Test-Klasse jetzt so verändert, dass sie grundsätzlich abgleichen, ob die gespeicherte `StoredCompany` gleich der retrieveten ist.
Dadurch sollte gewährleistet sein, dass alles abgeglichen wird.